src/containers/ServiceAccount/ServiceAccount.js

```diff
@@ -209,7 +209,13 @@ export /* istanbul ignore next */ class ServiceAccountContainer extends Componen
     let rowsForImgPull = [];
     if (serviceAccount.imagePullSecrets && !loading) {
       rowsForImgPull = serviceAccount.imagePullSecrets.map(({ name }) => {
-        const { type } = secrets.find(secret => secret.name === name);
+        const foundSecret = secrets.find(secret => secret.name === name);
```

Comment:
We could also write this as:
`const { type } = secrets.find(secret => secret.name === name) || { type: 'unknown-type' };`

Reply:
Yeah, this is nicer 👍